src/utils/trimTitle.js

```diff
@@ -0,0 +1,7 @@
+const trimTitle = (titleText) => {
+    let trimmedTitle = "";
+    trimmedTitle = titleText.length > 50 ? titleText.substring(0, 45) + "..." : titleText;
+    return trimmedTitle;
+}
```

Comment:
appreciation - liked that you're using a function to trim the string

suggestion - you can also use `text-overflow: ellipsis` to show the same output. reference - https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
Yes, this can be used when our text overflows the container. In my case, I wanted to trim the title and the title was not overflowing the card.